inc/patterns/hidden-heading-and-bird.php

```diff
@@ -16,6 +16,6 @@
 					<!-- /wp:group -->
 
 					<!-- wp:image {"align":"full","sizeSlug":"full","linkDestination":"none"} -->
-					<figure class="wp-block-image alignfull size-full"><img src="' . esc_url( get_stylesheet_directory_uri() ) . '/assets/images/flight-path-on-transparent-c.png" alt="' . esc_attr__( 'Illustration of a bird flying.', 'twentytwentytwo' ) . '"/></figure>
+					<figure class="wp-block-image alignfull size-full"><img src="' . esc_url( get_template_directory_uri() ) . '/assets/images/flight-path-on-transparent-c.png" alt="' . esc_attr__( 'Illustration of a bird flying.', 'twentytwentytwo' ) . '"/></figure>
```

Comment:
I know the other patterns don't break, but can you think of any other patterns where it would make sense to update the image path to use `get_template_directory_uri`?

Maybe it makes sense to update all of the dynamic URLs to use `get_template_directory_uri`, to be safe.

Reply:
Yeah, I'm not entirely sure. Twenty Twenty-Two uses `stylesheet_directory_uri` so I think that is technically a fine way to do it? I'd be interested in some insight from other theme devs too.